sdk/tests/deqp/framework/opengl/simplereference/sglrGLContext.js

```diff
@@ -127,15 +127,15 @@ goog.scope(function() {
          * @return {number}
          */
         var getWidth = function() {
-            return context.drawingBufferWidth;
+            return viewport[2];
```

Comment:
Thanks for pointing this. What about check viewport before using it? If viewport exists, return viewport size otherwise report drawingBuffer size.